client/js/main.js

```diff
@@ -124,23 +124,32 @@ function setScale() {
   roomRectangle = $('#roomArea').getBoundingClientRect();
 }
 
-async function uploadAsset() {
-  return selectFile('BINARY').then(async function(file) {
-    const response = await fetch('/asset', {
-      method: 'PUT',
-      headers: {
-        'Content-type': 'application/octet-stream'
-      },
-      body: file.content
+async function uploadAsset(multipleCallback) {
+  if (typeof(multipleCallback) === "function") {
+    return selectFile('BINARY', async function _f (f) {
```

Comment:
Any reason why the function has a name? Should work just fine without one but doesn't really matter.